modules/scalacheck/shared/src/main/scala/weaver/scalacheck/CheckConfig.scala

```diff
@@ -1,12 +1,14 @@
 package weaver
 package scalacheck
 
+import org.scalacheck.rng.Seed
+
 case class CheckConfig(
     minimumSuccessful: Int,
     maximumDiscardRatio: Int,
     maximumGeneratorSize: Int,
     perPropertyParallelism: Int,
-    initialSeed: Option[Long]
+    initialSeed: Option[Seed]
```

Comment:
I noticed that this was already implemented in https://github.com/disneystreaming/weaver-test/pull/633, which also future-proofs the `CheckConfig` datatype.

Perhaps we should retarget that instead?

Reply:
The future-proofing of `CheckConfig` would be greatly appreciated. Feel free to retarget it need be (or copy and credit the original author)